.gitignore

```diff
@@ -2,3 +2,6 @@ vendor
 _site
 .sass-cache
 .jekyll-metadata
+
+# vim
+*.sw[a-p]
```

Comment:
shouldn't you add this to your `.gitignore_global`?

Reply:
True